pyr2.html

```diff
@@ -58,31 +56,38 @@
 var a13 = qnbank[Qline][7];
 var snod = qnbank[Qline][8];
 var a14 = qnbank[Qline][9];
-document.write("<DL>");
+
+document.innerHTML="<DL>";
 document.write("<table border=0 width=550>");
-document.write("<tr>");
+document.innerHTML="<tr>";
 document.write("<td width=100% align=left>");
-document.write("<DT>", qn);
-document.write("<br><br>");
-document.write("<DD>",snoa,'. ', a11);
+document.innerHTML="<p> Burger           </p>";
+document.innerHTML="<DT>";
+document.write(qn);
```

Comment:
Avoid ever using `document.write` and always use DOM to create, move, and manipulate HTML elements.

See:
- http://stackoverflow.com/questions/802854/why-is-document-write-considered-a-bad-practice
- http://stackoverflow.com/questions/4537963/what-are-alternatives-to-document-write